examples/jsm/lines/LineGeometry.js

```diff
@@ -15,19 +15,13 @@ class LineGeometry extends LineSegmentsGeometry {
 	setPositions( array ) {
 
 		// converts [ x1, y1, z1,  x2, y2, z2, ... ] to pairs format
+		const count = array.length / 3 - 1;
+		const points = new Float32Array( 6 * count );
 
-		const length = array.length - 3;
-		const points = new Float32Array( 2 * length );
+		for ( let i = 0; i < count; i ++ ) {
 
-		for ( let i = 0; i < length; i += 3 ) {
-
-			points[ 2 * i ] = array[ i ];
-			points[ 2 * i + 1 ] = array[ i + 1 ];
-			points[ 2 * i + 2 ] = array[ i + 2 ];
-
-			points[ 2 * i + 3 ] = array[ i + 3 ];
-			points[ 2 * i + 4 ] = array[ i + 4 ];
-			points[ 2 * i + 5 ] = array[ i + 5 ];
+			points.set( array.slice( 3 * i, 3 * ( i + 1 ) ), 6 * i );
+			points.set( array.slice( 3 * ( i + 1 ), 3 * ( i + 2 ) ), 6 * i + 3 );
```

Comment:
I'd prefer to revert these changes and keep the PR focused on any changes related to opacity. I find it harder to read and it will be slower and generate more garbage with the unnecessary "slice" calls.

Reply:
revert to previous synthax

Reply:
It looks like there's still a pretty significant diff in the LineGeometry.js file. I don't think it needs to be touched at all for the changes being proposed.

Reply:
completly reverted